docker-compose.yml

```diff
@@ -17,5 +26,15 @@ services:
       - ./frontend:/app
       - /app/node_modules
     ports:
-      - "3000:3000"
+      - "4000:4000"
```

Comment:
Just out of curiosity, why does the frontend port need to be switched to `4000`? If this change does go through, we need to make sure to update the readme and the contributing guidelines :)

Reply:
will change this to 3000